From a93ca63d01e7cd1e40b5be576992f77fac364bd5 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 21 Mar 2016 18:21:17 +0200
Subject: [PATCH] gstreamer-gl.pc.in: don't append GL_CFLAGS to CFLAGS

Dependencies' include directories should not be added in this way;
it causes problems when cross-compiling in sysroot environments.

Upstream-Status: Pending

[Updated to apply to 1.12]

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 pkgconfig/gstreamer-gl.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gst-plugins-bad-1.12.1/pkgconfig/gstreamer-gl.pc.in
===================================================================
--- gst-plugins-bad-1.12.1.orig/pkgconfig/gstreamer-gl.pc.in
+++ gst-plugins-bad-1.12.1/pkgconfig/gstreamer-gl.pc.in
@@ -10,4 +10,4 @@ Version: @VERSION@
 Requires: gstreamer-base-@GST_API_VERSION@ gstreamer-@GST_API_VERSION@
 
 Libs: -L${libdir} -lgstgl-@GST_API_VERSION@
-Cflags: -I${includedir} -I${libdir}/gstreamer-@GST_API_VERSION@/include @GL_CFLAGS@
+Cflags: -I${includedir} -I${libdir}/gstreamer-@GST_API_VERSION@/include
